app/assets/stylesheets/comp-content.scss

```diff
@@ -0,0 +1,276 @@
+.content {
+
+  @mixin circle($size) {
+    color: $color_text;
+    font-weight: 400;
+    border: 1px solid #00909E;
+    border-radius: 100%;
+    font-size: 0.775em;
+    text-align: center;
+    margin-right: 0.25em;
+    width: $size;
+    height: $size;
+    line-height: $size;
+    display: inline-block;
+  }
+
+  .gradient-bg {
+    background-image: linear-gradient(-180deg, #ecfafc 0%, white 50%);
+    margin-bottom: 5em;
+
+    .inner {
+      max-width: 650px;
+      margin: 0 auto;
+      padding: 1em;
+      padding-top: 5em;
+
+      @media (min-width: 768px) {
+        padding: 0;
+        padding-top: 5em;
+      }
+
+      h1 {
+        font-weight: 600;
+        font-size: 2.25em;
+        margin-bottom: 0.25em;
+      }
+      p {
+        font-weight: 300;
+        color: $color_main;
+        font-size: 1.25em;
+        margin-top: 0;
+        line-height: 1.3;
+      }
+      ul {
+        list-style: none;
+        padding-left: 0;
+      }
+    }
+  }
+
+  article {
+    text-align: center;
+
+    & > * {
+      text-align: left;
+      margin: 1em auto;
+      max-width: 630px;
+    }
+
+    & > img {
+      text-align: left;
+      margin: 1em auto;
+      max-width: 100%;
+    }
+
+    p {
+      font-size: 1.125em;
+      line-height: 1.5;
+    }
+  }
+
+  ol {
+    counter-reset: list;
+    list-style: none;
+    font-size: 1.125em;
+    padding-left: 0;
+
+    li {
+      counter-increment: list;
+      margin-bottom: 1em;
+    }
+    li:before {
+      @include circle(18px);
+      content: counter(list);
+      margin-right: 8px;
+
+      @media (min-width: 768px) {
+        margin-left: -29px;
+      }
+    }
+  }
+
+  ul li {
+    margin-bottom: 0.25em;
+  }
+
+  .note {
+    position: relative;
+    padding-top: 0.5em;
+    margin-bottom: 4em;
+    font-size: 0.775em;
+    color: $text_soft;
+
+    p:before {
+      content: ' ';
+      position: absolute;
+      top: 0.5em;
+      width: 6em;
+      height: 1px;
+      background: rgba($text_soft, .2);
+    }
+  }
+
+  h2 {
+    font-size: 1.875em;
+    padding-bottom: 0.5em;
+    border-bottom: 1px solid #DEEFF1;
+  }
+
+  h3 {
+    margin-top: 2em;
+
+    small {
+      @include circle(18px);
+    }
+  }
+
+  h4 {
+    font-weight: 600;
+    text-transform: uppercase;
+    margin-top: 2em;
+  }
+
+  h5 {
+    font-size: 1em;
+  }
+
+  hr {
```

Comment:
To make them bigger than the container I had to add have specific margin rules, so they aren't as global as they could be. I would like to unify or reuse though.